packages/seo-store/package.json

```diff
@@ -14,7 +20,9 @@
     "watch": "yarn clean && babel src --watch --out-dir build",
     "clean": "rimraf build",
     "lint": "eslint src --max-warnings=0",
-    "lint:watch": "esw src --max-warnings=0 --watch"
+    "lint:watch": "esw src --max-warnings=0 --watch",
+    "test": "jest",
+    "test:watch": "npm test -- --watch"
```

Comment:
Same as on the other PR: let's use Yarn instead
```suggestion
    "test:watch": "yarn test -- --watch"
```

Or even directly, or do you expect params in the `test` script?

```suggestion
    "test:watch": "jest --watch"
```